### PR TITLE
4.1.1

### PR DESCRIPTION
**Pull request by PikaDude**
Fixes various issues regarding pitches of notes with the town tune editor and player, specifically removing the odd note F1 which caused most issues.

### CODE REVIEW

**Review by Julian-MM — APPROVED** [review]
When testing the /develop/ branch and comparing it to the /master/ branch, the town tunes definitely sound off-key in the /master/ branch, and they definitely sound a lot more like they should in the /develop/ branch.

I've looked through my code additions again and I don't notice any errors, and when testing out the aspects of the extension this patch should fix, they now all work as expected.

**Comment by PikaDude** [comment]
I was waiting for an approval from someone else other than the code author before merging :/ Try not to do this again.

**Comment by Julian-MM** [comment]
I thought that you would want that, but you had specifically only requested a review from me, so despite thinking it was odd, I assumed that if you wanted more collaborators approval you would have requested it with the feature specifically built for communicating that. 

I reviewed it and saw no further activity for about an entire day, so given the unusually important nature of the fix, and the continued influx of bug reports, I merged the seemingly fulfilled PR, assuming you were waiting for the code author to do so after all requirements were met. Still learning GitEtiquette after all, so sorry about misreading that.

Because this I don't see this _entirely_ as my fault, but I definitely agree it was rushed of me, and I'll be much more careful about merging PR's next time.

**Pull request merged**
